qgispluginci/__about__.py

```diff
@@ -47,13 +54,34 @@
 __uri_tracker__ = f"{__uri__}issues/"
 
 # This string might be updated on CI on runtime with a proper semantic version name with X.Y.Z
-__version__ = "__VERSION__"
+__version__ = getenv("GITHUB_REF_NAME", "__VERSION__")
 
 if "." not in __version__:
     # If __version__ is still not a proper semantic versioning with X.Y.Z
     # let's hardcode 0.0.0
     __version__ = "0.0.0"
 
+    # try to get it from git tag
+    try:
+        git_cmd = "git tag -l --sort=-creatordate | head -n 1"
+        returned_output: bytes = subprocess.check_output(git_cmd, shell=True)
+        __version__ = returned_output.decode("utf-8")
+    except Exception as err:
+        logging.debug(f"Unable to retrieve version from latest git tag. Trace: {err}")
+
+    if not len(__version__):
```

Comment:
```suggestion
    if __version__ == "0.0.0":
```
Otherwise, you have the changelog parser.